### PR TITLE
stack: add exec command

### LINKED ISSUE #1770: page improve: stack
Improve page for [The Haskell Tool Stack](https://docs.haskellstack.org/en/stable/README/). There is existed but contain a very limited manual content.

### PR DESCRIPTION
**Pull request by wisn**
Fix #1770

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/stack.md [hunk 1]
```diff
@@ -25,3 +25,7 @@
 - Compile a project and execute a command after compilation:
 
 `stack build --exec "{{command}}"`
+
+- Run a program and pass an argument to it:
```
Comment (sbrl) [inline pages/common/stack.md]: Is this a haskell program that it runs?

Comment (wisn) [reply pages/common/stack.md]: @sbrl Yes. It will run a program from `./stack-work` directory.

Comment (sbrl) [reply pages/common/stack.md]: Cool, thanks for the clarification!  Just wants to check I'm on the same page :-)

**Comment by sbrl** [comment]
Thanks, @wisn!

**Pull request merged**